src/AccelStepperFirmata.cpp

```diff
@@ -97,7 +97,7 @@ boolean AccelStepperFirmata::handleSysex(byte command, byte argc, byte *argv)
       if (stepCommand == ACCELSTEPPER_CONFIG) {
         interface = argv[index++];
         wireCount = (interface & 0x70) >> 4; // upper 3 bits are the wire count
-        stepType = (interface & 0x07) >> 1; // bits 4-6 are the step type
```

Comment:
The mask is right here but the comment is wrong. It should read: `// bits 1-3 are the step type`.

According to the protocol definition: 
Bit 0 is for the enable pin. 
Bits 1-3 indicate the step type. 
Bits 4-6 indicate the wire count.

Reply:
Actually you're right, the mask should be 0x0e for bits 1-3. Both the mask and the comment were wrong.

Reply:
Haha, cool. I was worried for a second there that I had no clue what I was talking about, since I am definitely not a computer scientist